components/OrderCard.tsx

```diff
@@ -9,7 +9,7 @@ export const OrderCard = ({ order }) => {
         <div className="flex-1">
           <div className="text-md font-bold">ID: {order.id}</div>
           <div>{order.order_text}</div>
-          <div>{order.ordered_at}</div>
+          <div>{order.ordered_at.toLocaleString()}</div>
```

Comment:
currently ordered_at is returned as a string, but that would probably be inaccurate to display as we are storing it as a date object in the db, which is in utc time. probably need to define a type for orders, and use that for the supabase query, then call .toLocaleString() on the date object returned